examples/kubernetes-in-docker/0_export_env_vars.sh

```diff
@@ -26,8 +26,8 @@ export TEST_APP_LOADBALANCER_SVCS="${TEST_APP_LOADBALANCER_SVCS:-false}"
 # pulled by KinD.
 #
 # These should be configured/customized in customize.env
-export CREATE_DOCKER_INTERNAL_REGISTRY="${CREATE_DOCKER_INTERNAL_REGISTRY:-true}"
-if [[ "CREATE_DOCKER_INTERNAL_REGISTRY" == "false" ]]; then
+export USE_DOCKER_LOCAL_REGISTRY="${USE_DOCKER_LOCAL_REGISTRY:-true}"
+if [[ "USE_DOCKER_LOCAL_REGISTRY" == "false" ]]; then
```

Comment:
I *think* we need the `else` condition for this:
```
else
  export DOCKER_REGISTRY_URL=localhost:5000
fi
```
and the export in 1_create_kind_cluster needs to be deleted:
```
  export DOCKER_REGISTRY_URL="${reg_ip}:${reg_port}"
```
I think we want localhost for this (only the KinD config gets the container IP, right)?